starter-code/views/random.hbs

```diff
@@ -0,0 +1,19 @@
+<main id="beer-list">
+  <div class="beer-card">
+    <div class="beer-thumb-container">
+      <img class="beer-thumb" src="{{randomBeer.[0].image_url}}" />
+    </div>
+    <div>
+      <h2>{{randomBeer.[0].name}}</h2>
```

Comment:
¿Esto te funciona?

Reply:
Hola ! Sí !

![image](https://user-images.githubusercontent.com/3327056/75090393-2642e000-556b-11ea-993e-86ce50222d2a.png)

Vi eso:
https://stackoverflow.com/questions/8044219/how-do-i-access-an-access-array-item-by-index-in-handlebars

Igualmente mejor que pase solo el record individual al view